cli/replicator_delete.go

```diff
@@ -31,9 +31,8 @@ func MakeReplicatorDeleteCommand(cfg *config.Config) *cobra.Command {
 	)
 	var cmd = &cobra.Command{
 		Use:   "delete [-f, --full | -c, --collection] <peer>",
-		Short: "Delete a replicator",
-		Long: `Use this command if you wish to remove the target replicator
-	for the p2p data sync system.`,
+		Short: "Delete a replicator. It will stop synchronizing",
+		Long:  `Remove a replicator. It will stop synchronizing.`,
```

Comment:
Nitpick: one says remove and one says delete

Reply:
✅